frontend/src/lib/constants.tsx

```diff
@@ -258,6 +258,7 @@ export const FEATURE_FLAGS = {
     SHOW_NEW_EXPERIMENTATION_ENGINE_BANNER: 'show-new-experimentation-engine-banner', // owner: @andehen #team-experiments
     WEB_ANALYTICS_FRUSTRATING_PAGES_TILE: 'web-analytics-frustrating-pages-tile', // owner: @lricoy #team-web-analytics
     SQL_EDITOR_AI_ERROR_FIXER: 'sql-editor-ai-error-fixer', // owner: @Gilbert09 #team-data-warehouse
+    GET_TEMPLATES_FROM_DB: 'get-templates-from-db', // owner: @meikel #team-cdp
```

Comment:
I'm being very nitpicky but surely slightly more descriptive? We have a lot of things that are "templates" in posthog
```suggestion
    GET_HOG_TEMPLATES_FROM_DB: 'get-hog-templates-from-db', // owner: @meikel #team-cdp
```